### ISSUE TITLE
No-op for cljs 

### ISSUE THREAD

**Issue by sundbry** [plumatic/schema#377]
Hello,
First, thank you for this library, I have been using it in projects for two years now.

I would like to publish release builds with the schema code *completely* compiled out - I am trying to shave the milliseconds off of my startup time (running cljs on mobile) and my CPU profiler is full of calls to schema, with-meta, conditional-schema constructors, etc. With each function or data structure that gets added, it potentially adds a good portion of a millisecond to the launch. I don't know if any prior work has been attempted to this extreme?

**Comment by w01fe** [comment]
Thanks for the kind words! Have you seen the section of the README about
compile-fn-validation?  I haven't used it for this purpose, but I believe
you should be able to set this flag at cljs compile time to prevent
generation of all schema function validation code at macroexpansion time.
Is that what you're looking for?

**Comment by sundbry** [comment]
Hi @w01fe, I gave that a try yesterday (by disabling *assert*), or at least I thought I did, and it didn't change the generated code size. Upon further inspection it just seems really finnicky trying to turn off *assert* in the cljs compiler like @hlship mentioned in #336. Experimenting more, I manually set-compile-fn-validation! to false and it finally delivered a significant performance impact (about 25% faster startup time / ~20% smaller code size). So I think that will be good enough. Thanks.
